bin/lint

```diff
@@ -2,6 +2,7 @@
 
 bin/bundle exec rubocop --autocorrect-all $*
 yarn prettier --write --list-different --ignore-unknown ${*:-'**/*'}
+bin/bundle exec rufo --check app
```

Comment:
Does rufo take cmdline args? Can we add `$*`?

Reply:
Yeah